server/test/auth/CiviFormProfileMergerTest.java

```diff
@@ -62,9 +62,10 @@ public void setup() {
 
     account.setApplicants(Collections.singletonList(applicant));
 
+    when(applicant.getAccount()).thenReturn(account);
```

Comment:
question (non-blocking): Just curious why you moved this?

Reply:
The Applicant object needs to have an Account before the Application is created on line 67, otherwise there will be a null exception when getAccount() is called in the Application constructor. This is only an issue for tests since in reality the Applicant will always have an associated Account.

Reply:
ah gotcha. thanks for explaining!